ammo-fe/src/util/Config.ts

```diff
@@ -0,0 +1,13 @@
+interface IConfig {
+    beUrl: string;
+    port: number;
+}
+
+const buildConfig = (): IConfig => {
+    return {
+        beUrl: process.env.REACT_APP_BE_URL,
+        port: parseInt(process.env.PORT, 10),
+    };
+};
+
+export default buildConfig();
```

Comment:
@Spartakyste  I followed your advice and tried to use functions and not classes when not needed! :)

Reply:
@GaelFerrand Could be simplified ! 
const buildConfig = (): IConfig => ({
         beUrl: process.env.REACT_APP_BE_URL,
         port: parseInt(process.env.PORT, 10),
     });;